identity-apps-core/apps/authentication-portal/src/main/webapp/oauth2_authz.jsp

```diff
@@ -233,6 +248,45 @@
                             }
                         %>
 
+                        <%
+                            if (MapUtils.isNotEmpty(authorizationDetailsToBeDisplayed)) {
+                        %>
+                            <div style="text-align: left;">
+                                <div class="claim-list ui list">
+                                    <div class="item mt-2">
+                                        <i aria-hidden="true" class="circle tiny icon primary consent-item-bullet" id="Authorization Details Types"></i>
+                                        <div class="content mt-2">
+                                            <div class="header light-font">
+                                                <%=AuthenticationEndpointUtil.i18n(resourceBundle, "requested.authorization-details")%>
```

Comment:
Let's use the i18n function defined in identity-apps-core/apps/authentication-portal/src/main/webapp/includes/localize.jsp instead.